### ISSUE TITLE
Linux chrome driver downloads msedgedriver not avaliable for linux

### ISSUE THREAD

**Issue by senpl** [webdriverio/selenium-standalone#620]
## Expected Behavior
I just want to install chrome driver for linux.
## Current Behavior
It fails with msg:
`ENOENT: no such file or directory, chmod '/home/l/.npm-packages/lib/node_modules/selenium-standalone/.selenium/chromiumedgedriver/latest-x64/msedgedriver'`
I do not even want msedgedriver.
## Steps to Reproduce
selenium-standalone install --drivers.chrome.version=98.0.4758.80 --drivers.chrome.baseURL=https://chromedriver.storage.googleapis.com

1.  download google-chrome
2.  install selenium-standalone
3.  selenium-standalone install --drivers.chrome.version=98.0.4758.80 --drivers.chrome.baseURL=https://chromedriver.storage.googleapis.com

## Logs
```
Error in "chmod". 
See more details below:
ENOENT: no such file or directory, chmod '/home/l/.npm-packages/lib/node_modules/selenium-standalone/.selenium/chromiumedgedriver/latest-x64/msedgedriver'
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[Error: ENOENT: no such file or directory, chmod '/home/l/.npm-packages/lib/node_modules/selenium-standalone/.selenium/chromiumedgedriver/latest-x64/msedgedriver'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'chmod',
  path: '/home/l/.npm-packages/lib/node_modules/selenium-standalone/.selenium/chromiumedgedriver/latest-x64/msedgedriver'
}

Node.js v17.3.0
```

If there are lots of logs, please link to [a public gist](https://gist.github.com/).

## Your Environment
* Version of `selenium-standalone` that you are using: 8.0.9
* Is there another tool calling `selenium-standalone` on your behalf:no
* System/platform:Arch Linux 
* Other details that might be important:might be related to https://github.com/webdriverio/webdriverio/issues/8006 still it's not same, coz here driver probably not exist, and no known fix exist afaik

**Comment by christian-bromann** [comment]
Thanks for reporting!

Any contributions that resolves the bug are highly appreciated. Please take a look into our [contribution guidelines](https://github.com/webdriverio/selenium-standalone/blob/main/CONTRIBUTING.md) and let us know if you have any questions. Cheers!

**Comment by cshuiu** [comment]
I get the same error when I try to install chromium edge. It says the file doesn't exist, but it exists as a '.zip' file, 

`Error in "chmod". 
See more details below:
ENOENT: no such file or directory, chmod '[PATH]/node_modules/selenium-standalone/.selenium/chromiumedgedriver/98.0.1108.51-x64/msedgedriver'`

Config file
`  chromiumedge: {
    version: '98.0.1108.51',
    arch: process.arch,
    baseURL: 'https://msedgedriver.azureedge.net/'
  }`

![image](https://user-images.githubusercontent.com/97026622/154084862-776e002c-f5ed-45b5-9a14-a8fdafc1deb5.png)

**Comment by cshuiu** [comment]
I might have found the root cause of the issue. The `isUpToDate` function always returns true for MSEdge driver, which prevents installation later. So, the driver downloads, then checks if it has to be installed (though based on commends it should not download either) and since it thinks it's already the latest version, it skips installation.

```
// inside isUpToDate
  /**
   * the msedgedriver.azureedge.net endpoint to download edgedriver
   * doesn't support the "If-None-Match" header and will always re-download
   * it. To prevent this we always assume it is the latest
   */
  if (hash && url.includes('edgedriver')) {
    return cb(true);
  }
```

```
    // File already exists. Prevent download/installation.
    if (isLatest) {
      logger('---');
      logger('File from ' + opts.from + ' has already been downloaded');
      return;
    }
```

**Comment by christian-bromann** [comment]
@senpl thanks for investigating. I am not sure why we had this special case for edgedriver. Maybe a git blame tells more about it. Mind raising a PR with a proper change?

**Comment by senpl** [comment]
@christian-bromann > @senpl thanks for investigating. I am not sure why we had this special case for edgedriver. Maybe a git blame tells more about it. Mind raising a PR with a proper change?
I could try, still I currently get setDriverFilePermissions msgs from source and no action is performed. I will need to run it from source before submitting PR, which will take time.

**Comment by senpl** [comment]
OK. I fixed it on my machine. PR is comming, but I am not sure it this is correct fix so checks are welcomed.

**Comment by cshuiu** [comment]
@senpl Are you opening the PR? I also need this functionality fixed :)

**Comment by senpl** [comment]
check if this fixed issue. If not then provide test that creates expected condition coz this seams to fix build on my machine still I am not sure if this is the only needed fix.

**Comment by senpl** [comment]
Issue is fixed for me in version 8.0.10 Latest.
